.circleci/docker/Dockerfile.ubuntu1904

```
@@ -26,6 +26,8 @@ FROM buildpack-deps:disco AS base
 ARG DEBIAN_FRONTEND=noninteractive
 
 RUN set -ex; \
+	echo "deb http://ppa.launchpad.net/ethereum/cpp-build-deps/ubuntu disco main" >> /etc/apt/sources.list ; \
```

```suggestion
	echo "deb http://ppa.launchpad.net/ethereum/cpp-build-deps/ubuntu $(lsb_release -cs) main" >> /etc/apt/sources.list ; \
```
I think it's not bad to make this line a little more generic, to make future upgrades of ubuntu docker image versions less error prone. What do you think?

---

I like it - but sadly ``lsb_release`` won't be available here.
Maybe ``source /etc/lsb-release && echo "deb http://ppa.launchpad.net/ethereum/cpp-build-deps/ubuntu $DISTRIB_CODENAME main" >> /etc/apt/sources.list ; \``?

---

I'll change it in the PR - let me know if it works.

---

:-1: sadly.